modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion.cc

```diff
@@ -115,21 +115,21 @@ DiffusionSolver::Initialize()
 
   if (options.verbose) log.Log() << text_name_ << ": Global number of DOFs=" << num_global_dofs_;
 
-  opensn::mpi.Barrier();
+  opensn::mpi_comm.barrier();
   log.Log() << "Sparsity pattern";
-  opensn::mpi.Barrier();
+  opensn::mpi_comm.barrier();
```

Comment:
for later. @wdhawkins has mentioned several times that he believes we have too many barriers. This comes from a time when the old code had "training wheels". I believe we should make an issue to investigate whether we have excessive barriers. It's a to-do for after the cleanup phase.

Reply:
Opened #97 for this.